diff --git a/node_modules/@types/alpinejs/index.d.ts b/node_modules/@types/alpinejs/index.d.ts
index 1080f9e..483374e 100644
--- a/node_modules/@types/alpinejs/index.d.ts
+++ b/node_modules/@types/alpinejs/index.d.ts
@@ -150,7 +150,7 @@ interface XDataContext {
 }
 
 export interface Stores {
-    [key: string | symbol]: unknown;
+    [key: string | symbol]: any;
 }
 
 export interface Magics<T> {
@@ -365,7 +365,7 @@ export interface Alpine {
     // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
     data: <T_12 extends Record<string | symbol, unknown>>(
         name: string,
-        callback: (...args: unknown[]) => AlpineComponent<T_12>, // Needed generic to properly autotype objects
+        callback: (...args: any[]) => AlpineComponent<T_12>, // Needed generic to properly autotype objects
     ) => void;
     bind: (name: string | ElementWithXAttributes, bindings: Bindings | ((...args: unknown[]) => Bindings)) => void;
 }
