diff --git a/node_modules/astro/import-meta.d.ts b/node_modules/astro/import-meta.d.ts
index 23d951c..8f6395e 100644
--- a/node_modules/astro/import-meta.d.ts
+++ b/node_modules/astro/import-meta.d.ts
@@ -13,7 +13,7 @@ interface ImportMeta {
 
 	readonly env: ImportMetaEnv;
 
-	glob: import('vite/types/importGlob').ImportGlobFunction;
+	glob: import('vite').ImportGlobFunction;
 	/**
 	 * @deprecated Use `import.meta.glob('*', { eager: true })` instead
 	 */
